store/conf/attrs/zimbra-attrs.xml

```diff
@@ -5260,7 +5260,7 @@ TODO: delete them permanently from here
   <desc>enable/disable MAPI (Microsoft Outlook) Connector</desc>
 </attr>
 
-<attr id="1128" name="zimbraDumpsterEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountInfo,accountInherited,domainAdminModifiable,domainInherited" since="7.0.0">
+<attr id="1128" name="zimbraDumpsterEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountInfo,accountCosDomainInherited,domainAdminModifiable,domainInherited" since="7.0.0">
```

Comment:
No change in associated files? Like we run get-set steps to generate definitions?

Reply:
Had run `ant generate-getters` found no diff.